src/common/colors.js

```diff
@@ -49,4 +49,5 @@ export default {
   HEADER_COLOR_1_DARK: "#77baff",
   HEADER_COLOR_1_LIGHT: "#0066ff",
   HEADER_COLOR_2_LIGHT: "#727272",
+  BANIDB_LIGHT: "#eaa040",
```

Comment:
I saw some reference to colors being directly used in code, can you add those colors in this files as well.